Services/UI/OJS.Services.Ui.Business/Implementations/ParticipantsBusinessService.cs

```diff
@@ -41,6 +44,28 @@ public Task<int> GetPracticeParticipantsCount(int contestId)
     public Task<int> GetCompeteParticipantsCount(int contestId)
         => this.participantsData.GetAllByContestAndIsOfficial(contestId, true).CountAsync();
 
+    public async Task UpdateTotalScoreSnapshotOfParticipants() =>
+       await this.participantsData.UpdateTotalScoreSnapshot();
+
+    public async Task RemoveParticipantMultipleScores()
+    {
+        var participantScores =
+            this.scoresDataService.GetAll()
+                .GroupBy(ps => new { ps.IsOfficial, ps.ProblemId, ps.ParticipantId })
+                .Where(ps => ps.Count() > 1)
+                .ToList();
+
+        var participantScoresToRemove = new List<ParticipantScore>();
+        foreach (var participantScoreGroup in participantScores)
+        {
+            participantScoresToRemove
+                .AddRange(participantScoreGroup.OrderByDescending(ps => ps.Points).Skip(1)
```

Comment:
create a separate variable for the input:
`participantScoreGroup.OrderByDescending(ps => ps.Points)
                                      .Skip(1)`

Also, fix the the indentation:
`participantScoresToRemove.AddRange()
                                              .ToList()`;